packages/parser/css_parser.mly

```diff
@@ -46,42 +47,51 @@ stylesheet:
   ;
 
 stylesheet_without_eof:
-  rs = list(rule) { (rs, Lex_buffer.make_loc $startpos $endpos) }
+  rs = list(with_whitespace(rule)) { (rs, Lex_buffer.make_loc $startpos $endpos) } // with whitespace also add a lot of warnings here
```

Comment:
> with whitespace also add a lot of warnings here

Don't fully understand this comment

Reply:
I meant menhir warnings, it increased the number of shitf/reduce errors a lot